### ISSUE TITLE
[Bug] iOS LinearGradientBrush Transparent Color

### ISSUE THREAD

**Issue by AlleSchonWeg** [xamarin/Xamarin.Forms#13378]
### Description
Using this sample xaml:
```
<ContentPage.Content >
		<StackLayout Spacing="0">

			<Frame BorderColor="White"
       HeightRequest="120"
       WidthRequest="120">
				<Frame.Background>
					<LinearGradientBrush EndPoint="0,1">
						<GradientStop Color="Transparent"
                          Offset="0.2" />
						<GradientStop Color="White"
                          Offset="1.0" />
					</LinearGradientBrush>
				</Frame.Background>
			</Frame>
		</StackLayout>

	</ContentPage.Content>
```
shows on iOS a gray area. ;)
### Expected Behavior
All should be white. Like android.

### Actual Behavior
Shows a gray area.

### Basic Information

- Version with issue: 4.8.0.1560
- Last known good version:
- Platform Target Frameworks: 
  - iOS:  
  - Android:  
  - UWP:   
- Android Support Library / AndroidX Version: 
- NuGet Packages:
- Affected Devices:

### Environment

<details><summary>Show/Hide Visual Studio info</summary>

```
```

</details>

### Screenshots
iOS:
![ios_Brush](https://user-images.githubusercontent.com/6727366/104332141-2683fb00-54f0-11eb-9e30-dde724d50d7c.png)

Android:
![android_Brush](https://user-images.githubusercontent.com/6727366/104332274-4b786e00-54f0-11eb-9ec2-3dbf8ecd9183.PNG)

### Workaround
Use skia

**Comment by Coding-BookMark** [comment]
i have the same issue in List View Items in Gradient Frame Control

**Comment by AlleSchonWeg** [comment]
@jsuarezruiz
Could you merge this into SR 4?

**Comment by Painwraith** [comment]
This is still a problem.

**Comment by Paul-N** [comment]
It is not fixed in latest release (5.0.0.2291). Check out demo here: https://github.com/Paul-N/LinearGradientBrushBugDemo Please reopen
